### PR TITLE
Update of the French version of Open Vpn GUI

### PR DESCRIPTION
**Pull request by bubuleur**
Hello,
I just put my contribution for the French translation of OpenVpn Gui
Can you put it in the next update
In thanking you
(The next update is for when?)

### CODE REVIEW

**Comment by mattock** [comment]
I asked if someone on openvpn-devel could review this. We might push out a new release in a few weeks, but nothing has been decided yet.

**Comment by bubuleur** [comment]
Thank you I look forward to this update

**Comment by bubuleur** [comment]
Re
To find out you have someone to review my translations
cordially
Thank you

**Comment by mattock** [comment]
Yes, Eric Crist said he had some feedback. I pinged him again.

**Comment by bubuleur** [comment]
Hello,
I come back to you just to know what stage is the update?
Thank you.

File: res/openvpn-gui-res-fr.rc [hunk 1]
```diff
@@ -75,12 +75,12 @@ END
 /* Challenge Response Dialog */
 ID_DLG_CHALLENGE_RESPONSE DIALOG 6, 18, 212, 72
 STYLE WS_SIZEBOX | WS_POPUP | WS_VISIBLE | WS_CAPTION | DS_CENTER | DS_SETFOREGROUND
-CAPTION "OpenVPN - Challenge Response"
+CAPTION "OpenVPN"
```
Comment (mattock) [inline res/openvpn-gui-res-fr.rc]: Any particular reason to omit translating "Challenge Response" in this dialog?

Comment (bubuleur) [reply res/openvpn-gui-res-fr.rc]: C'est une erreur de ma part oui cette ligne ne devrait pas etre traduite
Merci

Comment (mattock) [reply res/openvpn-gui-res-fr.rc]: @bubuleur : if you revert this particular change then I think this PR is good to go. My wife took a quick look at the translations and they looked good.

**Comment by bubuleur** [comment]
It is modified thanks for everything just to know you know how long there will be a new update thank you

**Comment by mattock** [comment]
Looks good, will merge now. We have not scheduled the next OpenVPN (or Windows installer) release yet, but I think it won't take too much time. Meanwhile you can use the [snapshot installers](https://build.openvpn.net/downloads/snapshots/) if you dare. The next build after merging this should have the latest openvpn-gui code in it, including your language changes.

**Comment by bubuleur** [comment]
Ok thank you very much for everything
Have a good day

**Pull request merged**